src/App.js

```diff
@@ -1,12 +1,53 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./Components/PokemonCard";
+
+
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokeArmy, setPokeArmy] = useState([])
+    const [currentUrl, setCurrentUrl] = useState(`https://pokeapi.co/api/v2/pokemon?limit=20&offset=0`)
+    const [nextUrl, setNextUrl ] = useState(``)
+    const [previousUrl, setPreviousUrl] = useState(``)
+
+
+    useEffect(()=>{
+        const abortCont = new AbortController()
+    async function fetchPokemons(){
+        try{ const response = await axios.get(currentUrl,{signal:abortCont.signal})
+            console.log(response)
+            setNextUrl(response.data.next)
+            setPreviousUrl(response.data.previous)
+            setPokeArmy(response.data.results)
+        } catch (e) {
+            console.error(e)
+        }
+    }
+    fetchPokemons()
+    return()=> abortCont.abort();
+    },[currentUrl])
+
+
+    function nextPage(){
+        setCurrentUrl(nextUrl)
+    }
+    function previousPage(){
+        setCurrentUrl(previousUrl)
+    }
+    return(
+        <>
+            <button onClick={previousPage && previousPage}>Previous</button>
```

Comment:
Het is beter om de button te disablen als previousPage null of undefined is
```suggestion
            <button disabled={!previousUrl} onClick={previousPage}>Previous</button>
```